frontend/public/components/operator-lifecycle-manager/install-plan.tsx

```diff
@@ -95,7 +95,7 @@ export type InstallPlanTableRowProps = {
 
 export const InstallPlansList = requireOperatorGroup((props: InstallPlansListProps) => {
   const EmptyMsg = () => <MsgBox title="No Install Plans Found" detail="Install Plans are created automatically by subscriptions or manually using the CLI." />;
-  return <Table {...props} aria-label="Install Plans" Header={InstallPlanTableHeader} Row={InstallPlanTableRow} EmptyMsg={EmptyMsg} />;
+  return <VirtualTable {...props} aria-label="Install Plans" Header={InstallPlanTableHeader} Row={InstallPlanTableRow} EmptyMsg={EmptyMsg} />;
```

Comment:
This table was not virtualized before. I know we had a problem with that in the past for some OLM tables where we explicitly disabled it to fix some bugs.

@alecmerdler is this change OK?

Reply:
Should be fine.